README.md

```diff
@@ -20,7 +20,9 @@ and your node app will run.
 
 ## How does it work?
 
-You need this POM as pom.xml in your nodejs project:
+You'll need a ```server.js``` file to be the entrypoint of your app by default.  (To make jarnode point elsewhere, set a ```NODE_ENTRY_FILENAME``` environment variable)
```

Comment:
wouldn't it be better to just read package.json for the "main"?

I mean, this might be ok... but I don't see why it's a better idea than just a boring default. package.json is where it is, right?

Reply:
Yep agree with that